src/components/common/PostContent.jsx

```
@@ -0,0 +1,38 @@
+    return null;
+  }
+
+  const contentToRender = stripImages
+    ? post.content.replace(/<img[^>]*>/g, "")
+    : post.content;
+
+  return (
+    <div className="post-content">
+      <div
+        className="post-content-body"
+        dangerouslySetInnerHTML={{ __html: contentToRender }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)